packages/@react-aria/calendar/docs/useCalendar.mdx

```diff
@@ -458,6 +458,14 @@ The `isReadOnly` boolean prop makes the Calendar's value immutable. Unlike `isDi
 <Calendar aria-label="Event date" value={today(getLocalTimeZone())} isReadOnly />
 ```
 
+### Custom first day of week
+
+By default, the first day of the week is Sunday. This can be changed by setting the `firstDayOfWeek` prop to a number between 0 (Sunday) and 6 (Saturday).
```

Comment:
i thought the API was going to be `firstDayOfWeek: 'Sun' | 'Mon' | ...`?

Reply:
The reasoning was to more match the Intl API: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getWeekInfo#browser_compatibility

But seems like the number may be confusing. Updated!